src/grt/src/fastroute/src/utility.cpp

```diff
@@ -339,12 +303,28 @@ void FastRouteCore::fillVIA()
 
         int l1 = treenodes[node1].botL;
         int l2 = treenodes[node2].botL;
-        auto [bottom_layer, top_layer] = std::minmax(l1, l2);
+        int bottom_layer = std::min(l1, l2);
+        int top_layer = std::max(l1, l2);
```

Comment:
I went with separete function because with std::minmax we used the `auto` definition for the variables, and that made the variables `const`. Down in the code it is necessary to change the variables so I made it to be separate function calls.

Reply:
Also I couldn't find what structure the `auto` was using, so separating was the solution I found.